lib/probes/mongodb.js

```diff
@@ -0,0 +1,348 @@
+'use strict'
+
+const shimmer = require('ximmer')
+const semver = require('semver');
+
+const ao = require('..')
+const requirePatch = require('../require-patch');
+const conf = ao.probes.mongodb;
```

Comment:
Q: Is ao.probes.mongodb the same as ao.probes['mongodb']?

Reply:
yes.